src/course-home/mentoring-tab/MentoringTab.jsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { injectIntl, intlShape } from '@edx/frontend-platform/i18n';
+
+import ChatWindow from './ChatWindow';
+
+import messages from './messages';
+import './MentoringTab.scss';
+
+const MentoringTab = ({ intl }) => (
+  <>
+    <h2 aria-level="1" className="h2 my-3">
```

Comment:
[nit]: I wonder if it is necessary to change `aria-level="1"` for the `<h2>` heading? By default, all headers have `aria-level` equal to the header level (`<h2>` - `aria-level="2"`), in this case you increase `aria-level` (`<h2>` - `aria-level="1"`).

I [looked](https://app.pr-1758-49e51b.sandboxes.opencraft.hosting/learning/course/course-v1:DEMO+TC01+TC01/dates), on other tabs (Progress, Dates) a similar aria attribute is added in case of using another tag, different from the heading (`<div role="heading" aria-level="1" class="h2 my-3">Important dates</div>`).

Let's change this a `<h1>` level heading and remove `aria-level`

```suggestion
    <h1 className="my-3">
```